scripts/types-mom/panels.d.ts

```diff
@@ -1,10 +1,13 @@
+import { uint64 } from 'common/web';
+
```

Comment:
Oh I completely missed this, I see what's happened. This type is exported from the website repo, where we use bigints for uint64s, not strings, I should probably remove it from here. `uint64_str` is probably what you want here, or `steamID`.